app/models/job_proxy_dispatcher.rb

```diff
@@ -90,6 +91,36 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def dispatch_container_jobs
+    jobs_to_dispatch, = Benchmark.realtime_block(:pending_container_jobs) { pending_jobs(ContainerImage) }
+    Benchmark.current_realtime[:container_jobs_to_dispatch_count] = jobs_to_dispatch.length
+
+    concurrent_container_scans_limit = Settings[:container_scanning][:concurrent_per_ems].to_i
+    if jobs_to_dispatch.length > 0
+      jobs_to_dispatch.each do |job|
+        if concurrent_container_scans_limit > 0 &&
+           active_scans(ContainerImage)[@zone] >= concurrent_container_scans_limit
```

Comment:
Is the first conditional needed?

If `concurrent_container_scans_limit` is 0
and nothing is happening, won't 
`active_scans(ContainerImage)[@zone] >= concurrent_container_scans_limit`

Become:
`0 >= 0`

Therefore, you'd get the warning and break?